roofit/roofitcore/inc/RooFit/TestStatistics/RooBinnedL.h

```diff
@@ -15,8 +15,11 @@
 
 #include <RooFit/TestStatistics/RooAbsL.h>
 #include "RooAbsReal.h"
+#include <RooChangeTracker.h>
```

Comment:
```suggestion
```
You don't need this include in the `.h` if the change tracker member is only a pointer (and, if you use `unique_ptr` the destructor of the likelihood class is defined in the `.cc` file, even if it's trivial).